Mathlib/Data/IsROrC/Attr.lean

```diff
@@ -0,0 +1,17 @@
+/-
+Copyright (c) 2023 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+
+import Lean
+
+/-!
+# Declare a `isROrC_simps` simp attribute.
+
+Has to be in a separate file from `Data.IsROrC.Basic`, since simp attributes cannot be used directly
+in the file that declares them.
+-/
+
+/-- "Simp attribute for lemmas about `IsROrC`" -/
+register_simp_attr isROrC_simps
```

Comment:
Can we deliberately mis-name this as `is_R_or_C_simps` to make the diff easier to read? We can then rename it in a subsequent PR.

Reply:
Oh nevermind, that's not the problem